### PR TITLE
Schema compose extension taskinfo dev preview

### PR DESCRIPTION
**Pull request by clearab**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[b61ba4d](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/b61ba4d050d61f6bb8c5ef454845af9dcfb2b0bf)_: 

:clock10: **Preparing**: average preparing time is _45 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[b61ba4d](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/b61ba4d050d61f6bb8c5ef454845af9dcfb2b0bf)_: 

:clock10: **Incremental building**: average incremental building time is _20 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[b61ba4d](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/b61ba4d050d61f6bb8c5ef454845af9dcfb2b0bf)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ac-schema-compose-extension-taskinfo/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md)

<a name="msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md"></a>
### [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ac-schema-compose-extension-taskinfo/msteams-platform/resources/schema/manifest-schema-dev-preview.md)
 - **Line 10**: **[Warning]** Invalid file link:(~/resources/dev-preview/developer-preview-intro).
 - **Line 11**: **[Warning]** Invalid file link:(~/resources/schema/manifest-schema).
 - **Line 20**: **[Warning]** Invalid file link:(~/resources/dev-preview/developer-preview-features).
 - **Line 171**: **[Warning]** Invalid file link:(~/concepts/bots/bots-create).
 - **Line 221**: **[Warning]** Invalid file link:(~/concepts/apps/apps-package#icons).
 - **Line 283**: **[Warning]** Invalid file link:(~/concepts/bots/bots-menus).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C23%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201902230144036063-473%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LjhHiGdcOnZ3QnGRg6l38SjG%2B%2FBW271518sEYl28ATo%3D&st=2019-02-23T01%3A40%3A21Z&se=2019-03-26T01%3A45%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[96a9f6c](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/96a9f6cab5630f52927235fc890bf9a0aeaa5011)_: 

:clock10: **Preparing**: average preparing time is _45 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[96a9f6c](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/96a9f6cab5630f52927235fc890bf9a0aeaa5011)_: 

:clock10: **Incremental building**: average incremental building time is _20 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[96a9f6c](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/96a9f6cab5630f52927235fc890bf9a0aeaa5011)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ac-schema-compose-extension-taskinfo/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C23%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201902230147586933-473%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=g7soZf%2FJ7j14BbTGQQY7pnd%2F0B%2F%2BTxTMzA94PicZ794%3D&st=2019-02-23T01%3A44%3A07Z&se=2019-03-26T01%3A49%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ ms.date: 08/07/2018
 # Developer Preview Reference: version 1.3 manifest schema for Microsoft Teams
 
 > [!NOTE]
-> See [Developer Preview](~/resources/dev-preview/developer-preview-intro) for information on the program and how you can join.
-> If you are not using the developer preview you should not be using this version of the manifest. See [Reference: Manifest schema for Microsoft Teams](~/resources/schema/manifest-schema) for the public version of the manifest.
+> See [Developer Preview](~/resources/dev-preview/developer-preview-intro.md) for information on the program and how you can join.
+> If you are not using the developer preview you should not be using this version of the manifest. See [Reference: Manifest schema for Microsoft Teams](~/resources/schema/manifest-schema.md) for the public version of the manifest.
```
Comment (clearab) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md]: test comment

**Comment by o365devx** [comment]
OPS Build status updates of commit _[615297e](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/615297e18d88842db52e156de51553a35e33ab2f)_: 

:clock10: **Preparing**: average preparing time is _46 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[615297e](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/615297e18d88842db52e156de51553a35e33ab2f)_: 

:clock10: **Incremental building**: average incremental building time is _20 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[615297e](https://github.com/MicrosoftDocs/msteams-docs/pull/473/commits/615297e18d88842db52e156de51553a35e33ab2f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ac-schema-compose-extension-taskinfo/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C5%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201903052341222999-473%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DgLVUz49XEJUUO7PZDwcyO7yB%2FBeJ%2FxobbSDe6HwPqs%3D&st=2019-03-05T23%3A37%3A32Z&se=2019-04-05T23%3A42%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**